tool/task.dart

```diff
@@ -314,33 +314,42 @@ Future<Iterable<Map<String, Object?>>> _docFlutter({
 }) async {
   var flutterRepo = await FlutterRepo.copyFromExistingFlutterRepo(
       await cleanFlutterRepo, flutterPath, env, label);
-  await flutterRepo.launcher.runStreamed(
-    flutterRepo.cacheDart,
-    ['pub', 'get'],
-    workingDirectory: path.join(flutterPath, 'dev', 'tools'),
-  );
   try {
     await flutterRepo.launcher.runStreamed(
-      flutterRepo.cacheDart,
+      flutterRepo.dartCmd,
       ['pub', 'global', 'deactivate', 'snippets'],
     );
   } on SubprocessException {
     // Ignore failure to deactivate so this works on completely clean bots.
   }
   await flutterRepo.launcher.runStreamed(
-    flutterRepo.cacheDart,
+    flutterRepo.dartCmd,
     ['pub', 'global', 'activate', 'snippets'],
```

Comment:
We activate a particular version of snippets (0.4.0 at the moment) inside of the `dev/bots/docs.sh` script. Should you be activating the same one?